certified-connectors/Zippydoc/README.md

```diff
@@ -0,0 +1,18 @@
+## ZIPPYDOC Connector
+ZIPPYDOC allows to merge, transform, filter, extract, report and analyze structured and table based data and to flexibly create data flows. Our connector helps to integrate ZIPPYDOC with third party services.
+
+
```

Comment:
@worm333 please format the readme.md file according to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1). All of the required headings should be present.